src/assets/toolkit/styles/components/sky.css

```diff
@@ -238,15 +268,27 @@
  * 1. At large enough screens, make the menu _always_ visible.
  * 2. Remove the bump (since it won't be above anything at this point).
  * 3. Reset the background.
- * 4. Reset the order.
+ * 4. To align right, this now needs to be last.
+ * 5. Constraining the width keeps the last three links on a second line to avoid orphans.
  */
 
 @media (--Sky-nav-expanded-viewport) {
   .Sky-nav-menu {
     display: block; /* 1 */
     padding-bottom: 0; /* 2 */
     background: transparent; /* 3 */
-    order: 0; /* 4 */
+    order: 2; /* 4 */
```

Comment:
This was the crazy frustrating part. If, in the expanded view I had anything but the flex order of logo, `Sky-nav-controls` then nav, I couldn't get the nav to align right. This is because I'm using `justify-content: space-between` on `Sky-nav` to get the logo on the left and the nav on the right. Even with the third element is hidden, it still counts, so the nav wasn't aligning right. I've added comments to explain this, but probably not well enough!